config/locales/en.yml

```diff
@@ -16,6 +16,7 @@ en:
       short_name_with_naric: UK ENIC or NARIC
       full_name: UK ENIC (the UK agency that recognises international qualifications and skills)
       url: https://www.enic.org.uk
+  govuk_url: https://www.gov.uk
```

Comment:
Nit: we tend to nest the `url:` key under the service name, so we should do that here too:

```
govuk:
  url: https://www.gov.uk
```